pyproject.toml

```diff
@@ -72,7 +73,7 @@ truss = 'truss.cli:cli_group'
 mlflow = "^1.29.0"
 
 [build-system]
-requires = ["poetry-core>=1.0.0"]
+requires = ["poetry-core>=1.2.1"]
```

Comment:
General comment about changes to Truss's python dependencies, sometimes they conflict with those of Baseten client where it's a dependency, and it's easier to modify Truss deps to allow working with Baseten client. Would be good to check for that, in case you haven't done that already.